api/views.py

```
@@ -0,0 +1,101 @@
+from rest_framework import viewsets, status, generics
+from rest_framework.response import Response
+from rest_framework.decorators import action
+from django.utils import timezone
+from django.db.models import F
+from .models import (
+    Word, FlashcardGame, FlashcardReview, FlashcardStats
+)
+from .serializers import (
+    WordSerializer, FlashcardGameSerializer, FlashcardReviewSerializer,
```

Ruff (0.8.2)

`.serializers.WordSerializer` imported but unused

Remove unused import

(F401)